### ISSUE TITLE
Tebako does not build on Mac with the latest version of folly

### ISSUE THREAD

**Issue by d4rky-pl** [tamatebako/tebako#183]
Split from #177:

With folly `2024.08.05.00` installed via homebrew, `tebako setup` fails with:

```
/Users/semaphore/.tebako/deps/src/_dwarfs_wr/deps/src/_dwarfs/folly/folly/json/json.cpp:122:19: error: no member named 'double_mode' in 'folly::json::serialization_opts'
            opts_.double_mode,
            ~~~~~ ^
/Users/semaphore/.tebako/deps/src/_dwarfs_wr/deps/src/_dwarfs/folly/folly/json/json.cpp:124:19: error: no member named 'double_flags' in 'folly::json::serialization_opts'
 
            ~~~~~ ^
2 errors generated.
make[9]: *** [folly/CMakeFiles/folly_base.dir/folly/json/json.cpp.o] Error 1
make[9]: *** Waiting for unfinished jobs....
make[8]: *** [folly/CMakeFiles/folly_base.dir/all] Error 2
make[7]: *** [all] Error 2
make[6]: *** [/Users/semaphore/.tebako/deps/src/_dwarfs_wr/deps/src/_dwarfs-stamp/_dwarfs-build] Error 2
make[5]: *** [CMakeFiles/_dwarfs.dir/all] Error 2
make[4]: *** [all] Error 2
make[3]: *** [/Users/semaphore/.tebako/deps/src/_dwarfs_wr-stamp/_dwarfs_wr-build] Error 2
make[2]: *** [CMakeFiles/_dwarfs_wr.dir/all] Error 2
make[1]: *** [CMakeFiles/setup.dir/rule] Error 2
make: *** [setup] Error 2
Tebako script failed: 'tebako setup' build step failed [102]
```

**Comment by maxirmx** [comment]
@d4rky-pl 
I was able to reproduce this issue, and fixed it in gem 0,8.2
Thank you for reporting

**Comment by d4rky-pl** [comment]
@maxirmx I believe this one is also fixed and can be closed, thanks a lot for the support!
